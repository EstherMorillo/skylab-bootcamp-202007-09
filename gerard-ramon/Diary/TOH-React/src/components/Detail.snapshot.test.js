import React from 'react';
import renderer from 'react-test-renderer';
import Details from './Details';
import { BrowserRouter } from 'react-router-dom';

describe('Detail snapshot', () => {
	it('Should match', () => {
		const props = {
			match: {
				params: {
					heroId: '12',
					name: 'shuvi'
				}
			}
		};
		const tree = renderer.create(
			<BrowserRouter>
				<Details {...props} />
			</BrowserRouter>
		);
		expect(tree.toJSON()).toMatchSnapshot();
	});
});
