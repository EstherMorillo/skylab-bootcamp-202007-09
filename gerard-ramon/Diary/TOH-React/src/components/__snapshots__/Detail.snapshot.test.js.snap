// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Detail snapshot Should match 1`] = `
<div>
  <h2>
    Narco
     details
  </h2>
  <form>
    <p>
      id: 
      12
    </p>
    <label
      htmlFor="heroName"
    >
      name:
      <input
        name="heroName"
        onChange={[Function]}
        placeholder="Hero name"
        value="Narco"
      />
    </label>
  </form>
</div>
`;
